composer.json

```diff
@@ -24,7 +24,7 @@
         "friends-of-behat/mink-extension": "^2.5",
         "friends-of-behat/page-object-extension": "^0.3.2",
         "friends-of-behat/service-container-extension": "^1.1",
-        "sylius-labs/coding-standard": "^4.1.1",
+        "sylius-labs/coding-standard": ">=4.1.1, <=4.2.1",
```

Comment:
They changed something with their config file format, so I had to limit the version range to keep things working over the broad range of PHP versions tested here.